### DISCUSSION TITLE
Add ability to browse tags in repository screen.

### DISCUSSION THREAD

**Discussion by kapitoshka438** [community/community#114666] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.155.0 (126411063)
Device: iPhone 12 Pro Max (iOS 17.4.1)
</details>

It would be great to see tags not only releases in the repository window. Also, `change branch` feature should allow to switch to a tag as well. 
Many repos only work with tags and don't create releases. Especially those who don't have build assets.

**Comment by elisealix22** [comment]
Hi @kapitoshka438 , thank you for the thoughtful suggestions! I opened a feature request to browse tags and view code for a tag internally. Please let us know if you have any other feedback in the meantime!
